src/translations/nb.ts

```diff
@@ -185,16 +185,12 @@ const translations = {
   'stegvisning.afp.validation_error': 'Du må svare på om du har rett til AFP.',
   'stegvisning.ufoere.title': 'Uføretrygd og AFP (avtalefestet pensjon)',
   'stegvisning.ufoere.info':
-    'Du kan ikke kombinere uføretrygd og AFP. Du må velge før du fyller 62 år.',
+    'Før du fyller 62 år må du velge mellom å få AFP eller å beholde uføretrygden. {br}{br} AFP og uføretrygd kan ikke kombineres. Hvis du ikke gir oss beskjed, så beholder du uføretrygden og mister retten til AFP.',
```

Comment:
Forslag til alternativ tekst for siste setning:
"Hvis du ikke gir oss beskjed, mister du retten til AFP (men beholder uføretrygden)."

Begrunnelse:

1. Det viktigste (at man mister AFP) framheves.
2. En negativ konsekvens og en positiv konsekvens bør ikke forbindes med "og".